examples/jsm/objects/Reflector.js

```diff
@@ -256,11 +263,31 @@ Reflector.ReflectorShader = {
 
 		}
 
+		vec4 blur( sampler2D diffuse, float blurSize, vec4 uv ) {
```

Comment:
This performs just a single horizontal blur which unfortunately produces no good quality. You can easily see this in the mirror demo:

<img width="582" alt="image" src="https://github.com/mrdoob/three.js/assets/12612165/176f5b50-02f9-46c8-919e-e734ab04de5f">

#25839 added a more high quality blur (it does a vertical and horizontal blur + depth scaling) but unfortunately with massive performance hits).

So right, I think it's best to keep `Reflector` as it is and let projects decide on their own what kind of blur they implement depending on their requirements.